assets/src/edit-story/components/keyboard/index.js

```diff
@@ -53,7 +53,7 @@ const globalRef = createRef();
 
 function setGlobalRef() {
   if (!globalRef.current) {
-    globalRef.current = document;
+    globalRef.current = document.documentElement;
```

Comment:
@dvoytenko, I noticed that some events did not bubble all the way to the `document`, but only to the `documentElement`, so I changed this line here. Can you think of any drawbacks to this?

Reply:
In particular, a `Cmd+Option+1` shortcut pressed inside an input element only bubbled to the `documentElement`, not the `document`. However, if the same combo was pressed on a button element, it bubbled all the way to the `document`. I have no idea why!